.github/workflows/eslint_disable_check.py

```
@@ -0,0 +1,122 @@
+    """
+    Recursively check TypeScript files for eslint-disable statements in the 'src' directory.
+
+    Args:
+        directory (str): Path to the directory.
+
+    Returns:
+        bool: True if eslint-disable statement is found, False otherwise.
+    """
+    eslint_found = False
+
+    for root, dirs, files in os.walk(os.path.join(directory, 'src')):
```

Ruff (0.7.0)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)